src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$card-square: 98px;
+$card-gapNborder: 1px;
+$month-length: 28 + 1;
+$start-day: monday;
+
+$week-days:
+'Monday' '1',
+'Tuesday' '2',
+'Wednesday' '3',
+'Thursday' '4',
+'Friday' '5',
+'Saturday' '6',
+'Sunday' '7';
+
```

Comment:
i think you should use short versions of the days

"Implement start-day modifier for the calendar with mon, tue, wed, thu, fri, sat and sun values"